pallets/block-reward/src/lib.rs

```diff
@@ -284,7 +343,7 @@ pub mod pallet {
 
 		fn on_nonzero_unbalanced(amount: NegativeImbalanceOf<T>) {
 			let value = amount.peek();
-			Self::distribute_imbalances(amount, Event::<T>::TransactionFeesDistributed(value));
+			TokenLocker::<T>::mutate(|lock| *lock += value);
```

Comment:
I'm a little confused about why we need to have TokenLocker here. Would you mind to explain that?